wordpress-mobile/WordPress-Android
# Issue/udpate author location

### Fix
Update the author filter from the right/end to the left/start of the tab layout on the ***Blog Posts*** screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/9689.  The author filter image does not fall into the 16dp left/start keyline intentionally.  It was requested to vertically align the center of the author filter image with the center of the back arrow icon above it.  Also, the touch target for the author filter was expanded to fill the width of the view and match the height of the tabs.  See the screenshots and animation below for illustration.

![post_list_author](https://user-images.githubusercontent.com/3827611/56623775-9058c480-65ea-11e9-9dac-56939c625a5e.png)

![post_list_author_60](https://user-images.githubusercontent.com/3827611/56623793-9ea6e080-65ea-11e9-972a-92d2695598d6.gif)

### Test
0. Set app language to English.
1. Go to ***Sites*** tab.
2. Tap ***Blog Posts*** under ***Publish*** section.
3. Notice location of author filter in tab layout.
4. Long-press author filter.
5. Notice touch background.
6. Scroll ***Published***, ***Drafts***, ***Scheduled***, and ***Trashed*** tabs left/right.
7. Notice tabs scroll behind author filter with gradient between views.
8. Tap back arrow.
9. Go to ***Me*** tab.
10. Tap ***App Settings*** item.
11. Tap ***Interface Language*** item.
12. Tap ***Hebrew*** option.
13. Tap ***OK*** button.
14. Tap back arrow.
15. Repeat steps 1 through 7.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
